content/docs/introduction/roadmap.md

```diff
@@ -25,17 +25,19 @@ Here's a snapshot of what we're working on now:
 
 ### Observability
 
-* **Autoscaling visualizations**
+* **Autoscaling charts in Project Dashboard**
 
-    Adding dynamic charts, populated with real-time data, to help you visualize how autoscaling impacts CPU and RAM usage over time. See [Monitoring autoscaling](/docs/guides/autoscaling-guide#monitor-autoscaling) for a sneak peek.
+    Keeping with our improvements to monitoring and observability in Neon, our new [Autoscaling graphs](/docs/guides/autoscaling-guide#monitor-autoscaling) &#8212; available now when you edit a branch's compute instance &#8212; will soon be added to the Project Dashboard as well.
 
-### Developer workflows
+* **New Monitoring page**
+
+   We are also working on adding a new Monitoring page to the Neon Console that will display key Posgres, connection, and resource usage metrics in a dashboard.
```

Comment:
```suggestion
   We are also working on adding a new Monitoring page to the Neon Console that will display key Postgres, connection, and resource usage metrics in a dashboard.
```

Reply:
The "Improvements to Vercel" item has been delivered but there might be a couple of other items we can add in its place.